avdevice/plugin.yaml

```diff
@@ -3,23 +3,162 @@ plugin:
+            en: "If the plugin can not connect to the device it retries this often. This is especially useful for TCP connections on devices that are plugged into a switchable socket as most receivers need about 40-50 seconds to boot their network device."
+            
+    secondstokeep: 
+        type: int
+        default: 50
+        description:
+            de: "Dauer in Sekunden, wie lange ein Kommando, das nicht erfolgreich war in einem Zwischenspeicher aufbewahrt werden soll. Dies ist besonders bei TCP Verbindungen mit Geräten an schaltbaren Steckdosen relevant, da diese ofmals 30-40 Sekunden zum Hochfahren benötigen."
+            en: "Seconds the plugin should temporarily save a command to retry later on after establishing a connection. This is especially useful for TCP connections on devices that are plugged into a switchable socket as most receivers need about 40-50 seconds to boot their network device."
+            
+    responsebuffer:
+        type: int or boolean
+        default: '-5'
+        description:
+            de: "Schnell hintereinander empfangene Werte werden bei einer negativen Attributangabe im Puffer gespeichert und gemeinsam verarbeitet. Der Standardwert sollte dafür sorgen, dass keine Antworten verloren gehen. Einige Receiver antworten unter Umständen immer zuerst mit der Angabe zum auf dem Display gezeigten Wert. Der Puffer sorgt dafür, dass auch eine nachfolgende Rückmeldung evaluiert wird."
+            en: "Set this to a negative number to collect quickly received responses in a buffer and evaluate them collectively. The standard value should be fine and prevent responses getting lost. Some receivers might first respond to a command with an update of the display and then with the actual value. The buffer ensures the correct evaluation of the response."
     
+    autoreconnect: 
+        type: bool
+        default: False
+        description:
+            de: "Automatischer Versuch, sich mit dem Gerät zu verbinden, wenn keine Rückmeldung kommt oder die Verbindung verloren wurde. Dies sollte nicht notwendig sein, da das Plugin ohnehin bei jedem neuen Senden eines Befehls einen Verbindungsaufbauversuch startet."
+            en: "Automatically tries to reconnect if no response is received or connection is lost. This should not be necessary as the plugin always tries to reconnect before sending a command."
+
+    verboselevel: 
+        type: int
+        default: 0
+        description:
+            de: "Ein Wert zwischen 0 und 3, um das Verboselevel des Debugloggers zu definieren. Je höher, desto mehr Nachrichten werden geloggt."
+            en: "Value between 0 and 3 to define the verbose level of the debug logger. The higher the value, the more debug messages."
+  
 item_attributes:
-    # Definition of item attributes defined by this plugin
+    avdevice_zone[0-4]:
+        type: name[a-z]
```

Comment:
@msinn, what about the "regex-matching-attribute-name" feature (as mentioned in #81)? Could it be useful for numbered attribute names with same settings?

Further what is the type `name[a-z]`?